apps/rehabstod/src/components/GlobalAlert/GlobalAlert.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react'
+import { IDSAlertGlobal, IDSIconAttention } from '@frontend/ids-react-ts'
+
+export function GlobalAlert({ children }: { children: ReactNode }) {
```

Comment:
Consider adding tests that verifies that the correct things are beeing rendered by this component!

Reply:
Added test